app/stac_api/validators.py

```diff
@@ -364,8 +364,20 @@ def validate_geometry(geometry):
     return geometry
 
 
-def validate_item_properties_datetimes_dependencies(
-    properties_datetime, properties_start_datetime, properties_end_datetime
+def validate_datetime_format(date_string):
+    try:
+        if not isinstance(date_string, datetime) and date_string is not None:
```

Comment:
Just some nitpick: wouldn't it make sense to first check for None and then for the instance?